torchaudio/datasets/musdb_hq.py

```diff
@@ -0,0 +1,93 @@
+import os
+from pathlib import Path
+from typing import Tuple, Union, List, Optional
+
+import torch
+import torchaudio
+from torch import Tensor
+from torch.utils.data import Dataset
+from torchaudio import transforms as T
+
+_URL = "https://zenodo.org/record/3338373/files/musdb18hq.zip"
+_EXT = ".wav"
+
+
+class MUSDB_HQ(Dataset):
+    def __init__(
+        self,
+        root: Union[str, Path],
+        sources: List[str],
```

Comment:
It' not clear what kind of sources are required for this dataset. Could you give default values, for example, `_SOURCES = ["bass", "drums", "other", "mixture", "vocals"]`?